### PR TITLE
solved signout functionality

### LINKED ISSUE #2746: Signout link in Simulator Header is sending GET call instead of POST
**Describe the bug**
Signing Out from the simulator throws an error!

**To Reproduce**
1. Go to https://circuitverse.org/
2. sign in to your Account
3. Go to https://circuitverse.org/simulator
4. click on your name (right top in navbar)
5. try to signOut from there
6. throws an error 
(you were redirected to https://circuitverse.org/users/sign_out with the message, 404 OOPS, THE PAGE YOU ARE LOOKING FOR CAN'T BE FOUND! )

**Expected behavior**
User must be Sign Out normally without any error.

**Screenshots**

https://user-images.githubusercontent.com/76155456/147589287-4d9f57e1-a4df-4500-8dca-45d84080509b.mov

**Device Information [optional]:**
 - OS: macOS
 - Browser Chrome

### PR DESCRIPTION
**Pull request by khareyash05**
Fixes #2746 

Fixed signout functionality

### CODE REVIEW

**Review by tachyons — CHANGES_REQUESTED** [review]
This PR just redirects the user to home page. Use link_to helper instead: Ref: https://stackoverflow.com/questions/25659488/using-link-to-with-delete-action-in-rails

**Comment by khareyash05** [comment]
@tachyons So i just have to add this line `link_to 'Delete', manage_user_path(user), method: :delete` instead of `/` . RIght??

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -51,7 +51,7 @@
             <a class="dropdown-item" href="<%= user_groups_path(current_user) %>"><%= t("layout.header.my_groups") %></a>
             <a class="dropdown-item" href="<%= notifications_path(current_user) %>"><%= t("layout.header.notifications") %></a>
             <div class="dropdown-divider"></div>
-            <a class="dropdown-item" rel="nofollow" data-method="delete" href="<%= destroy_user_session_path %>"><%= t("sign_out") %></a>
+            <a class="dropdown-item" rel="nofollow" data-method="delete" href="<%= link_to "Logout", destroy_user_session_path, :method => :delete %>"><%= t("sign_out") %></a>
```
Comment (tachyons) [inline app/views/layouts/_header.html.erb:54]: Here you are trying to embed an acnhor tag inside anchor tag, try following this guide to get basic understanding about the rails syntax https://guides.rubyonrails.org/getting_started.html

**Comment by pulkit-30** [comment]
Hey @khareyash05 , 
Are you working on this issue?

**Comment by tachyons** [comment]
Closing due to inactivity

**Pull request closed without merging**